website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,215 @@
+
+The most common options are: `top`, `bottom`, `left`, `right`.
+
+![Placement examples of the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-placement.png)
+
+The Rich Tooltip supports additional placement options: `top-start`, `top-end`, `bottom-start`, `bottom-end`, `left-start`, `left-end`, `right-start`,and `right-end`.
+
+!!! Info
+
+The placement property is _relative_ to the element that toggles the component, rather than the direction of the pointer. E.g., `placement=left` will render the component to the _left_ of the toggle, but the pointer will be pointing to the right.
+!!!
+
+### Collision detection
+
+The Rich Tooltip supports collision detection, meaning the `placement` property might be overridden depending on the position of the component relative to the edge of the viewport. This ensures that the component doesn't extend outside of the viewport while still being associated with the toggle element.
+
+![Example of collision detection in the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-collision-detection.png)
+
+## Offset
+
+The default recommended distance between the toggle and the Rich Tooltip pointer is 4px.
+
+![Default spacing for the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-offset-spacing.png)
+
+!!! Info
+
+In some cases, adding an offset may be necessary to adjust the position of the RichTooltip. Changing the default offset should be done sparingly and only when it’s necessary to make sure that the component does not obscure or cover the toggle or other important information.
+!!!
+
+!!! Dont
+
+Don’t add extra offset if the Rich Tooltip would block important information, appear disconnected from the element it is meant to provide information for, or cause confusion for the user.
+
+![Example of adjusting the offset of the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-offset-dont.png)
+!!!
+
+## Interaction
+
+The Rich Tooltip supports two different interaction methods: a `hover/focus` ("soft") interaction and a `click` interaction. Which one to use depends on the type and complexity of content displayed by the component, whether _temporarily persisting_ the content in the UI is necessary or beneficial to the user, and whether the element toggling the Rich Tooltip is interactive or not.
+
+1. `Hover/focus` (default): displays when the mouse enters the toggle or the toggle receives focus. The `RichTooltip` has a timeout by default and will persist for 500 milliseconds (0.5 seconds) _after_ the mouse leaves the toggle, or the focus is removed from it.
+2. `On click`: displays when the user clicks the toggle with a mouse or if the toggle receives a keyboard event (`spacebar`, `enter/return`). If the content of the toggle element is interactive, an `on click` interaction _cannot_ be used.
+
+Independent of which interaction is used, the Rich Tooltip can be dismissed by clicking outside of the component or with the `esc` key.
```

Comment:
```suggestion
Independent of which interaction is used, the Rich Tooltip can be dismissed by clicking outside of the component or with the `ESC` key.
```

Keys are typically all caps in technical writing. Have we not been doing that? We should be consistent with ourselves but make a note for the backlog in case we haven't been doing this correctly.